examples/dreamcast/Makefile

```diff
@@ -12,11 +13,36 @@ ifdef KOS_CCPLUS
 	DIRS += cpp tsunami
 endif
 
-all:
-	for i in $(DIRS); do $(KOS_MAKE) -C $$i || exit -1; done
+ERROR_LOG = build_errors.log
 
-clean:
-	for i in $(DIRS); do $(KOS_MAKE) -C $$i clean || exit -1; done
+.PHONY: all clean dist show_errors clean_up
 
-dist:
-	for i in $(DIRS); do $(KOS_MAKE) -C $$i dist || exit -1; done
+all: show_errors clean_up
```

Comment:
This looks wrong. `make -j2` will then run `make -C $$i clean` and `make -C $$i` in parallel.

Reply:
Thanks!  I took care of it.